proposals/stage-1/TRAVEL_FUND/MEMBER_TRAVEL_FUND.md

```diff
@@ -0,0 +1,169 @@
+# OpenJS Foundation Membership Travel Fund
+
+## Purpose
+
+To establish and administer a fund for members of the OpenJS Foundation to travel and spread knowledge about and support the Javascript ecosystem and the Foundation.
+
+//TODO update link
+### [Current Year Fund](https://github.com/nodejs/admin/blob/master/TravelFunds/2019.md)
+
+//TODO update link
+### [2018 funds](https://github.com/nodejs/admin/blob/master/TravelFunds/2018.md)
+
+## Requirements
+
+* Candidates must be an individual member of one of the projects of the OpenJS Foundation. //TODO clarify?
+* Those requesting funds must indicate that they do not have funding available from another source, such as an employer or
+the event itself that might cover costs for presenters.
+
+## Process
+
+Requests must be made prior to the start of travel. Requests submitted once travel is complete will not be approved.
+
+It is strongly recommended that you submit your request far enough in advance to get approval before making relevant expenditures.
+Review of requests will take at least 72 hours and may take considerably longer. Note, with limited funds available, approval is not
+guaranteed. If a request is rejected, reimbursement will not be issued.The Foundation cannot pre-pay for travel expenditures.
+It is recommended to write a report after the trip.
```

Comment:
Will it make sense to link a sample report here?

Reply:
That makes sense. I was actually wondering if we should remove the report entirely, or else enforce it as a requirement. If you look at the [last year of funded travel](https://github.com/nodejs/admin/blob/master/TravelFunds/2019.md), nobody filled out a report.

Reply:
I'm +1 to removing the report

Reply:
Removed in latest commit.